src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -627,7 +627,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     
     
       <ItemGroup>
-        <_ResolvedCopyLocalPublishAssets Include="@(RuntimePackAsset)" Condition="'$(SelfContained)' == 'true'" />
+        <_ResolvedCopyLocalPublishAssets Include="@(RuntimePackAsset)"
```

Comment:
It is a timing issue? It is the same itemgroup but add at a different time?

Reply:
It's not a timing issue, it's that the same files are getting added twice when SelfContained and RuntimePackAlwaysCopyLocal are true, as they are included separately for each one.

See the description in the bug: https://github.com/microsoft/CsWinRT/issues/442#issuecomment-701157626